app/controllers/answers_controller.rb

```diff
@@ -0,0 +1,19 @@
+class AnswersController < ApplicationController
+    def create
+        @question = Question.find(params[:question_id])
+        @answer = @question.answer.new(answer_parmas)
```

Comment:
`s`は必要です。これはモデルで記述した`    has_many :answers` というメソッドを使用しているからです。
コンソールでデータを入れて確認をしてみるとよいでしょう。

```suggestion
        @answer = @question.answers.new(answer_parmas)
```